### DISCUSSION TITLE
Issues Converting VOB files after update

### DISCUSSION THREAD

**Discussion by savathuntechquestion** [mifi/lossless-cut#2092] [Support] [answered]
Hi after the new update on Windows 11 I'm having issues converting VOB files, in the past I converted them to .mp4 but now it throws an error (posted below) and the program recommends I convert them to mpeg but then there is no audio. The original VOB files play and sound fine. My goal in using Losslesscut is to convert and stitch together VOB files into 1 mp4 file for Plex with as little quality loss as possible.

The export settings I'm using:
Enable MOV Faststart (enabled)
Preserve all MP4/MOV metadata (conversion fails to MP4 whether enabled or disabled)
"avoid_negative_ts" (make zero option)
"ffmpeg" experimental flag (disabled)

Error: Command failed with exit code 1: D:\LosslessCut-win-x64\resources\ffmpeg.exe -hide_banner -i D:\In Progress Musicals\DVD Video\DVD Video\VTS_01_1 (7).VOB -map 0:1 -c:0 copy -map 0:2 -c:1 copy -map_metadata 0 -movflags +faststart -default_mode infer_no_subs -ignore_unknown -f mp4 -y D:\In Progress Musicals\DVD Video\VTS_01_1 (7)-00.00.00.000-00.24.35.007.mp4
Input #0, mpeg, from 'D:\In Progress Musicals\DVD Video\DVD Video\VTS_01_1 (7).VOB':
  Duration: 00:24:35.01, start: 0.300300, bitrate: 5823 kb/s
  Stream #0:0[0x1bf]: Data: dvd_nav_packet
  Stream #0:1[0x1e0]: Video: mpeg2video (Main), yuv420p(tv, bottom first), 720x480 [SAR 8:9 DAR 4:3], 29.97 fps, 29.97 tbr, 90k tbn
    Side data:
      cpb: bitrate max/min/avg: 7000000/0/0 buffer size: 1835008 vbv_delay: N/A
  Stream #0:2[0xa0]: Audio: pcm_dvd, 48000 Hz, stereo, s16, 1536 kb/s
[mp4 @ 0000021b9a65c380] Could not find tag for codec pcm_dvd in stream #1, codec not currently supported in container
[out#0/mp4 @ 0000021b9a65bd00] Could not write header (incorrect codec parameters ?): Invalid argument
[aost#0:1/copy @ 0000021b9a65db80] Error initializing output stream: 
Stream mapping:
  Stream #0:1 -> #0:0 (copy)
  Stream #0:2 -> #0:1 (copy)
    Last message repeated 1 times

    at makeError (D:\LosslessCut-win-x64\resources\app.asar\out\main\index.js:1800:13)
    at handlePromise (D:\LosslessCut-win-x64\resources\app.asar\out\main\index.js:2619:29)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

{
  "err": {
    "killed": false,
    "failed": true,
    "timedOut": false,
    "isCanceled": false,
    "exitCode": 1
  },
  "state": {
    "ffmpegExperimental": false,
    "preserveMovData": false,
    "movFastStart": true,
    "preserveMetadataOnMerge": false,
    "filePath": "D:\\In Progress Musicals\\DVD Video\\DVD Video\\VTS_01_1 (7).VOB",
    "fileFormat": "mp4",
    "externalFilesMeta": {},
    "mainStreams": [
      {
        "index": 0,
        "codec_name": "dvd_nav_packet",
        "codec_long_name": "DVD Nav packet",
        "codec_type": "data",
        "codec_tag_string": "[0][0][0][0]",
        "codec_tag": "0x0000",
        "id": "0x1bf",
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/90000",
        "start_pts": 27027,
        "start_time": "0.300300",
        "disposition": {
          "default": 0,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        }
      },
      {
        "index": 1,
        "codec_name": "mpeg2video",
        "codec_long_name": "MPEG-2 video",
        "profile": "Main",
        "codec_type": "video",
        "codec_tag_string": "[0][0][0][0]",
        "codec_tag": "0x0000",
        "width": 720,
        "height": 480,
        "coded_width": 0,
        "coded_height": 0,
        "closed_captions": 0,
        "film_grain": 0,
        "has_b_frames": 1,
        "sample_aspect_ratio": "8:9",
        "display_aspect_ratio": "4:3",
        "pix_fmt": "yuv420p",
        "level": 8,
        "color_range": "tv",
        "chroma_location": "left",
        "field_order": "bb",
        "refs": 1,
        "id": "0x1e0",
        "r_frame_rate": "30000/1001",
        "avg_frame_rate": "30000/1001",
        "time_base": "1/90000",
        "start_pts": 27027,
        "start_time": "0.300300",
        "duration_ts": 132750618,
        "duration": "1475.006867",
        "extradata_size": 150,
        "disposition": {
          "default": 0,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        },
        "side_data_list": [
          {
            "side_data_type": "CPB properties",
            "max_bitrate": 7000000,
            "min_bitrate": 0,
            "avg_bitrate": 0,
            "buffer_size": 1835008,
            "vbv_delay": -1
          }
        ]
      },
      {
        "index": 2,
        "codec_name": "pcm_dvd",
        "codec_long_name": "PCM signed 20|24-bit big-endian",
        "codec_type": "audio",
        "codec_tag_string": "[0][0][0][0]",
        "codec_tag": "0x0000",
        "sample_fmt": "s16",
        "sample_rate": "48000",
        "channels": 2,
        "channel_layout": "stereo",
        "bits_per_sample": 0,
        "initial_padding": 0,
        "id": "0xa0",
        "r_frame_rate": "0/0",
        "avg_frame_rate": "0/0",
        "time_base": "1/90000",
        "start_pts": 27027,
        "start_time": "0.300300",
        "duration_ts": 132738041,
        "duration": "1474.867122",
        "bit_rate": "1536000",
        "bits_per_raw_sample": "16",
        "disposition": {
          "default": 0,
          "dub": 0,
          "original": 0,
          "comment": 0,
          "lyrics": 0,
          "karaoke": 0,
          "forced": 0,
          "hearing_impaired": 0,
          "visual_impaired": 0,
          "clean_effects": 0,
          "attached_pic": 0,
          "timed_thumbnails": 0,
          "captions": 0,
          "descriptions": 0,
          "metadata": 0,
          "dependent": 0,
          "still_image": 0
        }
      }
    ],
    "copyStreamIdsByFile": {
      "D:\\In Progress Musicals\\DVD Video\\DVD Video\\VTS_01_1 (7).VOB": {
        "0": false,
        "1": true,
        "2": true
      }
    },
    "cutSegments": [
      {}
    ],
    "mainFileFormatData": {
      "filename": "D:\\In Progress Musicals\\DVD Video\\DVD Video\\VTS_01_1 (7).VOB",
      "nb_streams": 3,
      "nb_programs": 0,
      "format_name": "mpeg",
      "format_long_name": "MPEG-PS (MPEG-2 Program Stream)",
      "start_time": "0.300300",
      "duration": "1475.006867",
      "size": "1073739776",
      "bit_rate": "5823646",
      "probe_score": 26
    },
    "rotation": 360,
    "shortestFlag": false,
    "effectiveExportMode": "separate",
    "outSegTemplate": "${FILENAME}-${CUT_FROM}-${CUT_TO}${SEG_SUFFIX}${EXT}"
  },
  "platform": "win32",
  "version": "3.61.1"
}

**Comment by mifi** [comment, marked as answer]
hi. your vob file contains an audio track that is of type `pcm_dvd`. Losslesscut/ffmpeg is not able to mux this into a mp4 format. If you choose mpeg or mpegts as output format, maybe it works? i will add support in the next version so that when exporting to MOV format, losslesscut will automatically convert from `pcm_dvd` to a supported pcm codec (note that mp4 cannot handle pcm at all)

**Reply by savathuntechquestion** [reply]
Thanks for the explanation and converting to mpeg didn't throw an error but the file still has no audio. I appreciate the effort for next release, your program has helped me convert a lot of multipart VOB files into a single file for plex.
